.github/workflows/forge-test.yml

```
@@ -0,0 +1,56 @@
+name: Forge Test & Coverage
+
+on:
+  push:
+    paths:
```

actionlint (1.7.9)

could not parse as YAML: did not find expected '-' indicator

(syntax-check)